SECURITY.md

```diff
@@ -0,0 +1,68 @@
+# Security Policy
+
+This document describes the management of vulnerabilities for the
+Fastify project and its official plugins.
```

Comment:
We may want to change references to Fastify throughout this document.

Reply:
good spot